privacy-statement.md

```diff
@@ -0,0 +1,113 @@
+# Privacy Statement—English Wikipedia Account Creation Assistance
+Thank you for using the English Wikipedia Account Creation Assistance tool ("the Tool"). This Privacy Statement explains
+how we collect, use, and dispose of the data we collect from you, as required by the [Terms of Use of the Wikimedia
+Foundation Cloud Services ("WMCS") project][wmcs-tou].
+
+Except where otherwise noted, this Privacy Statement applies to all users of the English Wikipedia Account Creation
+Assistance project, including but not limited to:
+
+- Users who are using the Tool to request that Wikipedia accounts be created for them ("Requesters").
+- Established Wikipedia users who register and log in to the Tool in order to process account requests from Requesters
+  ("Account Creators"). Account Creators can be further divided into the following groups, though most are members of
+  any of these groups:
+  - "Administrators": Also called "Tool Admins", these are Account Creators who are very trusted and have extra
+    capabilities beyond what Account Creators ordinarily have, such as the ability to grant and revoke access to the
+    Tool. Note that this is not the same as, nor does it imply that a user is, a [Wikipedia
+    administrator][enwiki-admins].
+  - "Checkusers": Account Creators who also have the [CheckUser permission on Wikipedia][enwiki-cus] and are thus highly
+    trusted.
+  - "Stewards": Account Creators who are also [Stewards on Wikimedia projects][meta-stewards], and are likewise highly
+    trusted members of the community.
+  - "Tool Roots": These Account Creators are the projects leads and are ultimately responsible for the overall
+    management of the Tool. They are responsible for ensuring that the Tool is in compliance with Wikimedia Foundation
+    policies and guidelines and are the only users with access to the underlying WMCS infrastructure that the Tool runs
+    on.
+
+A full list of current Account Creators can be found on the [Tool's user list][acc-users], including which of the above
+groups they belong to. Account Creators are trusted, vetted members of the Wikipedia editor community who have been
+granted access to the Tool by the Administrators. They are required to be in good standing with the Wikimedia
+Foundation, to be 18 years of age or older, and to have signed the legally-binding [Wikimedia Foundation Confidentiality
+Agreement for Nonpublic Information][wmf-confidentiality-agreement]. Account Creators are subject to an activity
+requirement; if they do not use the Tool for 45 days, their access to the Tool is suspended.
```

Comment:
IIRC we increased this to 90. We should probably add a note to the tool code that this is now mentioned in this statement (which I can action)

```suggestion
requirement; if they do not use the Tool for 90 days, their access to the Tool is suspended.
```

Reply:
Ah yeah. The data purging script should also get a similar comment, since I reference the 15 day interval here too.